.changeset/breezy-windows-jump.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend': minor
+---
+
+The `callbackUrl` option of `OAuthAdapter` is now required.
```

Comment:
Am I being blind but I don't see this in the `api-report`? Or are we not exporting some things?